integration-tests/src/main/java/io/druid/testing/ConfigFileConfigProvider.java

```diff
@@ -0,0 +1,109 @@
+/*
```

Comment:
@rasahner I think this is old license header, can you replace the license headers in all places with the one at https://github.com/druid-io/druid/blob/20fdb627d99b3cf0d29c85270f801868d8c4d7c8/common/src/main/java/io/druid/metadata/PasswordProvider.java#L1 .